frontend/app-development/features/latestCommit/LatestCommit.test.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import { LatestCommit } from './LatestCommit';
+import { textMock } from '@studio/testing/mocks/i18nMock';
+import { createQueryClientMock } from 'app-shared/mocks/queryClientMock';
+import {
+  type ServicesContextProps,
+  ServicesContextProvider,
+} from 'app-shared/contexts/ServicesContext';
+import { queriesMock } from 'app-shared/mocks/queriesMock';
+
+const renderLatestCommit = (queries?: Partial<ServicesContextProps>) => {
+  const allQueries: ServicesContextProps = {
+    ...queriesMock,
+    ...queries,
+  };
+
+  render(
+    <ServicesContextProvider {...allQueries} client={createQueryClientMock()}>
```

Comment:
Here we can use renderwithproviders method. It already wraps the component in the servicecontextprovider with all necessary data 😊